Blocks/src/Controller/Admin/BlocksController.php

```diff
@@ -77,7 +71,7 @@ public function afterConstruct() {
  */
 	public function beforeFilter(Event $event) {
 		parent::beforeFilter($event);
-		$this->Security->unlockedActions[] = 'admin_toggle';
+		$this->Security->config('unlockedActions', 'admin_toggle');
```

Comment:
This is something that's still kinda strange because I don't know how this works in 3.0 I do know however that if this would work it would have to be ```toggle``` instead of ```admin_toggle```